Configuration/Generator/python/sherpa_ttbar_2j_MENLOPS_13TeV_MASTER_ExtGen_cff.py

```diff
@@ -29,7 +29,7 @@
 				" METS_BBAR_MODE 5;",
 				" NJET:=2; LJET:=2; QCUT:=20.;",
 				" ME_SIGNAL_GENERATOR Comix Amegic LOOPGEN;",
-				" OL_PREFIX={0} ".format(os.environ['CMS_OPENLOOPS_PREFIX']),
```

Comment:
@cms-sw/generators-l2 , aren't we missing a `;` here? Should not it be
```
" OL_PREFIX={0};".format(....)
```

Reply:
Hello @smuzaffar , thanks for catching it. Could you please add the ;? Thanks!!

Reply:
done